### ISSUE TITLE
stuck at length of mel chunks 0% every time for too long.

### ISSUE THREAD

**Issue by jedisun76cn** [numz/sd-wav2lip-uhq#75]
Using cuda for inference.
Reading video frames...
Number of frames available for inference: 200
(80, 402)
Length of mel chunks: 122
  0%|                                                                                            | 0/1 [00:00<?, ?it/s]
  0%|                                                                                            | 0/8 [00:00<?, ?it/s] 

<img width="594" alt="7af57d026cc728a173cabfc6d7924c4" src="https://github.com/numz/sd-wav2lip-uhq/assets/48370363/b5c29ed5-8fad-4413-8167-5f354e061a5f">

input video is 720* 1280，8 seconds，input audio file : wav file is 5 seconds. 
i do not know why it is so long to be stuck at mel chunks?   1800 seconds passed without any response.
how can we speedup the proc?  it is crazy slow!
i use laptop with GPU nividia 4070  8GB vRam.

**Comment by sjc57** [comment]
Try changing the resize factor to a larger number. If that doesn't help try changing to a different Stable Diffusion checkpoint.

**Comment by numz** [comment]
Hi,
Yes, you can try resize factor but your gpu seems good enough for processing 720x1280
Try  to usé -medvram when launching stable-diffusion-webui 
Let me know
Regards

**Comment by jedisun76cn** [comment]
how can i use " -medvram" ?  modified the code of "webui-user.bat"?

**Comment by numz** [comment]
Yes,
In command arg

**Comment by jedisun76cn** [comment]
i try to use “set COMMANDLINE_ARGS= --medvram”   ，but  it does not work, and still stuck for long!

**Comment by numz** [comment]
Just for check, try yo use resize factor x2 and Let me know the result please

**Comment by jedisun76cn** [comment]
yes,  last time i had set resize factor x2 and its speed can be acceptable! but the resolution of the output video file has also been halved.  So is there a way to  restore the resolution of the output video?

**Comment by numz** [comment]
You can upscale with this method : https://www.youtube.com/watch?v=3z4MKUqFEUk

But after doing It in resize factor x2 try x1 in same session

**Comment by jedisun76cn** [comment]
sorry , I don't understand what you meant("But after doing It in resize factor x2 try x1 in same session").
 i saw the youtube video ( https://www.youtube.com/watch?v=3z4MKUqFEUk) and i found it could be used to upscale the img not video that i output. 
how can i do with it?  Excuse me that  i am not familiar to operate with sd-ui.

**Comment by numz** [comment]
Sorry, i just want you make test with resize factor x2 and just after resize factor x1 to see if it still stuck.

True, but you can extract img from the final video, or you can active debug and found images in debug folder those with dst_xxxxx.png and use it in batch mode

**Comment by x0arthur** [comment]
The same issue. and on the webUI, each time there is an error in the video processing area.
